man/rgbasm-old.5

```diff
@@ -0,0 +1,333 @@
+Deprecated in 0.5.0, removed in 0.7.0.
+.Pp
+Macro arguments now handle quoted strings and parenthesized expressions as single arguments, so commas inside them are not argument separators and do not need escaping.
+.Pp
+Instead, just use commas without backslashes.
+.Ss '*' comments
+Deprecated in 0.4.1, removed in 0.5.0.
+.Pp
+Instead, use
+.Ql \&;
+comments.
+.Ss Nested macro definitions
+Removed in 0.4.2.
+.Pp
+Instead, put the nested macro definition inside a quoted string (making sure that none of its lines start with
+.Ic ENDM ) ,
+then interpolate that string.
+For example:
+.Bd -literal -offset indent
+MACRO outer
+    DEF definition EQUS """
+        MACRO inner
+            println (\1) - (\\1)
+        \enENDM"""
+    {definition}
+    PURGE definition
+ENDM
+    outer 10
+    inner 3 ; prints 7
+.Ed
+.Ss Negative DS
+Removed in 0.3.2.
+.Pp
+This was used to "rewind" the value of
+.Ic @
+in RAM sections, allowing labeled space allocations to overlap.
+.Pp
+Instead, use
+.Ic UNION .
+.Ss Treating multi-character strings as numbers
+Deprecated in 0.9.0.
+.Pp
+Instead, use a multi-value
+.Ic CHARMAP ,
+or explicitly combine the values of individual characters.
+.Ss rgbgfx -f/--fix and -F/--fix-and-save
+Removed in 0.6.0.
+.Pp
+Instead, use
+.Ql rgbgfx -c/--colors
+to explicitly specify a color palette.
+If using
+.Ql -c embedded ,
+arrange the PNG's indexed palette in a separate graphics editor.
+.Ss rgbgfx -D/--debug
+Removed in 0.6.0.
+.Sh REPLACED
+.Ss Defining constants and variables without DEF
+Deprecated in 0.7.0, removed in 0.8.0.
+.Pp
+.Ic EQU , EQUS , = , RB , RW ,
+and
+.Ic RL
+definitions used to just start with the symbol name, but had to be typed in column 1.
+.Pp
+Instead, use
+.Ic DEF
+before constant and variable definitions.
+Note that
+.Ic EQUS
+expansion does not occur for the symbol name, so you have to use explicit
+.Ql {interpolation} .
+.Ss Defining macros like labels
+Deprecated in 0.6.0, removed in 0.7.0.
+.Pp
+Macros used to be defined as
+.Ql name: MACRO .
+.Pp
+Instead, use
+.Ql MACRO name .
+.Ss Defining variables with SET
+Deprecated in 0.5.2, removed in 0.6.0.
+.Pp
+Variables used to be defined as
+.Ql name SET value .
+.Pp
+Instead, use
+.Ql DEF name = value .
+.Ss PRINTT, PRINTI, PRINTV, and PRINTF
+Deprecated in 0.5.0, removed in 0.6.0.
+.Pp
+These directives were each specific to one type of value.
+.Pp
+Instead, use
+.Ic PRINT
+and
+.Ic PRINTLN ,
+with
+.Ic STRFMT
+or
+.Ql {interpolation}
+for type-specific formatting.
+.Ss GLOBAL and XDEF
+Deprecated in 0.4.2, removed in 0.5.0.
+.Pp
+Instead, use
+.Ic EXPORT .
+.Ss HOME, CODE, DATA, and BSS
+Deprecated in 0.3.0, removed in 0.4.0.
+.Pp
+Instead of
+.Ic HOME ,
+use
+.Ic ROM0 ;
+instead of
+.Ic CODE
+and
+.Ic DATA ,
+use
+.Ic ROMX ;
+instead of
+.Ic BSS ,
+use
+.Ic  WRAM0 .
+.Ss JP [HL]
+Deprecated in 0.3.0, removed in ???.
+.Pp
+Instead, use
+.Ql JP HL .
+.Ss LDI A, HL and LDD A, HL
+Deprecated in 0.3.0, removed in ???.
```

Comment:
When were `LDI A, HL` and `LDD A, HL` removed?

Reply:
https://github.com/gbdev/rgbds/commit/d2a97e93, so:
```suggestion
Deprecated in 0.3.0, removed in 0.4.0.
```